lib/Raven/Legacy.php

```diff
@@ -14,7 +16,7 @@ class Raven_Client extends \Raven\Client
 {
 }
 
-/**
+/** @noinspection PhpDeprecationInspection
```

Comment:
I'm not sure to understand correctly what you mean. Also, anyway, why would you add this tag only to this class?